File: python/unidad_03/problema_3_02.py
'''
El programa imprime y suma los términos de una serie.
'''

def main():

    i = 2

    sumaSerie = 0

    varControl = True

    while (i <= 1800):

        sumaSerie += i
        print(f"Se suma el número: {i}")

        if varControl:

            i += 3
            print(f"Total: {sumaSerie} \n")

            varControl = False
            continue

        i += 2
        print(f"Total: {sumaSerie} \n")

        varControl = True

    print(f"Total final: {sumaSerie}")

if __name__ == '__main__':

    main()

File: java/unidad_03/Problema_3_02/src/Problema_3_02.java
/*
 * El programa imprime y suma los términos de una serie.
 */

public class Problema_3_02 {

    public static void main(String[] args) {

        int i = 2;

        int sumaSerie = 0;

        boolean varControl = true;

        while (i <= 1800) {

            sumaSerie += i;
            System.out.printf("Se suma el número: %d %n", i);

            if (varControl) {

                i += 3;
                System.out.printf("Total: %d %n %n", sumaSerie);

                varControl = false;
                continue;
            }
            i += 2;
            System.out.printf("Total: %d %n %n", sumaSerie);

            varControl = true;
        }
        System.out.printf("Total final: %d", sumaSerie);
    }
}